lib/media/ttml_text_parser.js

```diff
@@ -342,6 +347,24 @@ shaka.media.TtmlTextParser.addStyle_ = function(
       // dimension of the video. The flag is set to true when lines
       // are counted, and false otherwise.
       cue.snapToLines = false;
+
+      if (cue.region && cue.region instanceof shaka.polyfill.VTTCue.VTTRegion) {
```

Comment:
It would be much better to apply the region override in TextEngine after the parsers are done.  That way, the parser does not need to know anything about the polyfill and users of third-party parsers can still use your override config.